src/main/java/com/unascribed/fabrication/mixin/b_utility/legacy_command_syntax/MixinEntitySummonArgumentType.java

```diff
@@ -21,19 +22,19 @@
 @Mixin(RegistryEntryArgumentType.class)
 @EligibleIf(configAvailable="*.legacy_command_syntax")
 public class MixinEntitySummonArgumentType {
-	private boolean isNotEntityArgument = true;
+	private static final ThreadLocal<Boolean> fabrication$isNotEntityArgument = ThreadLocal.withInitial(() -> true);
```

Comment:
imo the ThreadLocal usage in `blinking_drops/MixinItemEntityRendererVanilla` is already kinda iffy.
(needs to be tested, but should probably be replaced)

but i'm pretty sure this would cause issues, did you test it?

since this would, set the bool per `Thread` not per `Object` in `<init>` ! 

this injection has gotten a lot harder todo in this version...